packages/clients/textLocator/src/polar-client.ts

```diff
@@ -0,0 +1,37 @@
+import client from '@polar/core'
+import packageInfo from '../package.json'
+import { addPlugins } from './addPlugins'
+import { services as layerConf } from './services'
+import { mapConfiguration } from './mapConfig'
+
+// eslint-disable-next-line no-console
+console.log(`TextLocator map client running in version ${packageInfo.version}.`)
+const containerId = 'polarstern'
+addPlugins(client)
+
+interface TextLocatorParameters {
+  urls: {
+    textLocatorBackend: string
+    gazetteerClient: string
+  }
+}
+
+export async function initializeClient({ urls }: TextLocatorParameters) {
+  client.rawLayerList.initializeLayerList(layerConf)
+  mapConfiguration.layerConf = layerConf
+  mapConfiguration.addressSearch = {
+    // @ts-expect-error | rest configured in addPlugins.ts (simple API)
+    searchMethods: [{ url: urls.gazetteerClient }],
+  }
+  // @ts-expect-error | local addition
+  mapConfiguration.geometrySearch = {
+    url: urls.gazetteerClient,
+  }
+  // @ts-expect-error | local addition
+  mapConfiguration.textLocatorBackendUrl = urls.textLocatorBackend
```

Comment:
How about https://github.com/Dataport/polar/pull/115/commits/a333ba41d2852c1d2d414d872c18082141bdaae2 ? The imported `client` is `any` anyway and I don't see any further use in creating an additional type since the object is not used again.